pkg/providers/version/version.go

```diff
@@ -32,7 +32,7 @@ const (
 	// Karpenter's supported version of Kubernetes
 	// If a user runs a karpenter image on a k8s version outside the min and max,
 	// One error message will be fired to notify
-	MinK8sVersion = "1.23"
+	MinK8sVersion = "1.25"
```

Comment:
Are there any other checks that we have in our E2E tests that do a check on 1.23? Off the top of my head, I recall that there are some AL2/AL2023 checks that we do on that version since we can't do an n-1 version on the AMIs

Reply:
Good call, missed those with the grep :(